src/main.rs

```diff
@@ -73,18 +74,11 @@ fn main() {
                 ),
         )
         .arg(
-            Arg::with_name("wasm-remote-insecure")
-                .env("CHIMERA_WASM_REMOTE_INSECURE")
-                .long("wasm-remote-insecure")
-                .takes_value(false)
-                .help("Do not verify remote TLS certificate. False by default"),
-        )
-        .arg(
-            Arg::with_name("wasm-remote-non-tls")
-                .env("CHIMERA_WASM_REMOTE_NON_TLS")
-                .long("wasm-remote-non-tls")
-                .takes_value(false)
-                .help("Wasm remote endpoint is not using TLS. False by default"),
```

Comment:
I removed these two arguments from the CLI interface: if present I think it would be even better if they can be set fine-grained per policy on the policies.yaml file -- defaulting them to the most secure option (tls + cert verification).

Reply:
Yes, as discussed let's remove it. I think it's better to handle that inside of a dedicated configuration file